arch/risc-v/src/mpfs/mpfs_clockconfig.c

```diff
@@ -42,11 +42,10 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#define OSC_FREQ 80000000UL
```

Comment:
Let's wait for clock configuration and ddr training to land in proper files (by  Eero), and then do the thing